flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/MapTypeTest.scala

```diff
@@ -85,13 +85,14 @@ class MapTypeTest extends MapTypeTestBase {
       "MAP[DATE '1985-04-11', TIME '14:15:16', DATE '2018-07-26', TIME '17:18:19']",
       "{1985-04-11=14:15:16, 2018-07-26=17:18:19}")
 
-    testAllApis(
+    testTableApi(
       map(valueLiteral(gLocalTime("14:15:16")), valueLiteral(localDateTime("1985-04-11 14:15:16")),
         valueLiteral(gLocalTime("17:18:19")), valueLiteral(localDateTime("2018-07-26 17:18:19"))),
-      "map('14:15:16'.toTime, '1985-04-11 14:15:16'.toTimestamp, " +
-          "'17:18:19'.toTime, '2018-07-26 17:18:19'.toTimestamp)",
+      "{14:15:16=1985-04-11 14:15:16, 17:18:19=2018-07-26 17:18:19}")
+
+    testSqlApi(
       "MAP[TIME '14:15:16', TIMESTAMP '1985-04-11 14:15:16', " +
-          "TIME '17:18:19', TIMESTAMP '2018-07-26 17:18:19']",
+        "TIME '17:18:19', TIMESTAMP '2018-07-26 17:18:19']",
```

Comment:
I find that we have a lot of this changes that remove the `toTimestamp` from the tests. 

I'm wondering could we align the `toTimestamp` with the SQL timestmap literal? 

Currently `"2018-07-26 17:18:19".toTimestamp` actually calls `"2018-07-26 17:18:19".cast(DataTypes.TIMESTAMP(3))`, however, according to the javadoc of `toTimestamp`, it should be the same with SQL `TIMESTAMP string`.

Reply:
Currently toTimestamp in Table API can't align to Timestamp literal, because it use `SqlTimeTypeInfo.TIMESTAMP` as the target type.

Reply:
There is no such thing in table API which align to Timestamp literal in SQL. I will add some comments to clarify this.